Tigran Kirakosov/flexbox_version/sass/_mixins.scss

```diff
@@ -0,0 +1,43 @@
+/*  MEDIA QUERY MANAGER
+
+0 - 600px:        Phone
+600 - 900px:      Tablet portrait
+900 - 1200px:     Tablet landscape
+[1200 - 1800px] is where our normal styles applies
+1800px + :        Big desctop
+
+$breakpoint arguement choices:
+- phone
+- tab-port
+- tab-land
+- big-desctop
+
+1em = 16px
+*/
+
+@mixin respond($breakpoint) {
+  @if $breakpoint == phone {
+    @media only screen and (max-width: 37.5em) {

```

Comment:
don't think that it's good idea - write `max-width: 37.5em` and after comment `//600 px` - it will be more readable if you just write width in `px` instead of `em`

Reply:
Okay, will describe width in px in MQs.